.github/workflows/release.yml

```
@@ -41,6 +41,21 @@ jobs:
         if: steps.skip_check.outputs.should_skip != 'true'
         with:
           submodules: 'recursive'
+          # fetch-depth of 0 means to fetch the entire history not just the latest commit,
+          # and this is necessary when a workflow is triggered by something other than pushing a tag
+          fetch-depth: 0
+
+      - name: Get previous tag
+        if: steps.skip_check.outputs.should_skip != 'true'
+        id: prev_tag
+        working-directory: ${{env.GITHUB_WORKSPACE}}
+        run: |
```

actionlint

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)